x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.test.ts

```diff
@@ -88,7 +88,7 @@ describe('benchmarks API', () => {
 
     const [config] = router.get.mock.calls[0];
 
-    expect(config.path).toEqual('/api/csp/benchmarks');
+    expect(config.path).toEqual('/internal/cloud_security_posture/benchmarks');
```

Comment:
_nit_ + not really related to this PR

i _think_ we should use the constants we defined in `constants.ts` in tests? unless i'm missing smth, that would be less error prone (no duplicated strings)

Reply:
I think it is a semantic discussion,
because then you might ask if the test is checking that the api is using the same constant? 
or verifying the actual address?
one might argue to check the constant's values as well, right?